grype/db/internal/gormadapter/open.go

```diff
@@ -9,16 +9,16 @@ import (
 	"github.com/anchore/sqlite"
 )
 
-var connectStatements = []string{
+var writerStatements = []string{
 	// performance improvements (note: will result in lost data on write interruptions).
 	// on my box it reduces the time to write from 10 minutes to 10 seconds (with ~1GB memory utilization spikes)
 	`PRAGMA synchronous = OFF`,
 	`PRAGMA journal_mode = MEMORY`,
 }
 
 // Open a new connection to a sqlite3 database file
-func Open(path string, overwrite bool) (*gorm.DB, error) {
-	if overwrite {
+func Open(path string, write bool) (*gorm.DB, error) {
```

Comment:
Nice! 

Since we're changing the signature naming from `overwrite` => `write` do we want to modify that in the callers as well? I think overwrite as a variable name is used a couple of calls up.

Otherwise LGTM